src/encoded/vis_defines.py

```diff
@@ -146,8 +146,8 @@
 HIC_FILE_TYPES = ['hic']
 BIGINTERACT_FILE_TYPES = ['bigInteract']
 VISIBLE_FILE_FORMATS = BIGBED_FILE_TYPES + BIGWIG_FILE_TYPES + HIC_FILE_TYPES + BIGINTERACT_FILE_TYPES
-VISIBLE_DATASET_TYPES = ["Experiment", "Annotation"]
-VISIBLE_DATASET_TYPES_LC = ["experiment", "annotation"]
+VISIBLE_DATASET_TYPES = ["Experiment", "Annotation", "Functional_Characterization_Experiment"]
```

Comment:
I think this should match the class name which doesn't have `_`, though it's hidden now and won't trigger any errors.